pandas/core/indexes/interval.py

```diff
@@ -393,25 +400,6 @@ def __contains__(self, key) -> bool:
         except KeyError:
             return False
 
-    @Appender(
-        _interval_shared_docs["to_tuples"]
-        % dict(
-            return_type="Index",
-            examples="""
-        Examples
-        --------
-        >>> idx = pd.IntervalIndex.from_arrays([0, np.nan, 2], [1, np.nan, 3])
-        >>> idx.to_tuples()
-        Index([(0.0, 1.0), (nan, nan), (2.0, 3.0)], dtype='object')
-        >>> idx.to_tuples(na_tuple=False)
-        Index([(0.0, 1.0), nan, (2.0, 3.0)], dtype='object')
-        """,
-        )
-    )
```

Comment:
By removing this, the docstring of `IntervalIndex.to_tuples` is now no longer correct (and misses the example).

The delegation mechanism will need a way to change docstrings?